libs/platform/core/src/services/token-resolver/default-token-resolver.service.ts

```diff
@@ -0,0 +1,40 @@
+import { resolve } from '@spryker-oryx/di';
+import { Observable, of } from 'rxjs';
+import {
+  TokenResolver,
+  TokenResolverService,
+  TokenResourceResolver,
+} from './token-resolver.service';
+
+const tokenRE = /^[A-Z]+\.[A-Z]+$/;
+
+export class DefaultTokenResolverService implements TokenResolverService {
+  protected resolvers = new Map<string, TokenResolver>();
+
+  protected isToken(resolver: string): boolean {
+    return tokenRE.test(resolver);
+  }
+
+  protected getResolverKey(tokenResolver: string): string {
+    return `${TokenResourceResolver}${tokenResolver}`;
```

Comment:
Ideally we will have an ability to provide multiple resolvers per namespace/domain. But it can be done in a followup, as this one is quite big anyway.